README.md

```diff
@@ -0,0 +1,80 @@
+# Terraform Module: Keybase Domain Proof
+
+> [Terraform](https://terraform.io/) Module for managing AWS Route 53 [DNS Records](https://docs.aws.amazon.com/workmail/latest/adminguide/add_domain.html) for a _Keybase Domain Proof_.
+
+## Table of Contents
+
+- [Terraform Module: Keybase Domain Proof](#terraform-module-keybase-domain-proof)
+  - [Table of Contents](#table-of-contents)
+  - [Requirements](#requirements)
+  - [Dependencies](#dependencies)
+  - [Usage](#usage)
+    - [Module Variables](#module-variables)
+    - [Module Outputs](#module-outputs)
+  - [Author Information](#author-information)
+  - [License](#license)
+
+## Requirements
+
+This module requires Terraform version `0.12.0` or newer.
+
+## Dependencies
+
+This module depends on a correctly configured [AWS Provider](https://www.terraform.io/docs/providers/aws/index.html) in your Terraform codebase.
+
+## Usage
+
+Add the module to your Terraform resources like so:
+
+```hcl
+module "keybase-domain-proof" {
+  source              = "operatehappy/route53-keybase-domain-proof/aws"
+  version             = "1.0.0"
+  zone_id             = "Z3P5QSUBK4POTI"
+  domain_proof        = "38834dez61"
+}
+```
+
+Then, fetch the module from the [Terraform Registry](https://registry.terraform.io/modules/operatehappy/route53-keybase-domain-proof) using `terraform get`.
+
+### Module Variables
+
+Available variables are listed below, along with their default values:
+
+| variable       | type   | description                            | default |
+|----------------|--------|----------------------------------------|---------|
+| `zone_id`      | string | ID of the DNS Zone to store Records in |         |
+| `record_ttl`   | string | TTL for all DNS records                | `300`   |
+| `domain_proof` | string | Keybase Domain Proof Record            |         |
+
+Additionally, the following variables are generated as [locals](https://www.terraform.io/docs/configuration/locals.html):
+
+| key            | value                                                  |
+|----------------|--------------------------------------------------------|
+| `zone_name`    | `data.aws_route53_zone.zone.name`                      |
+| `domain_proof` | "keybase-site-verification=${var.verification_record}" |
+
+### Module Outputs
+
+Available outputs are listed below, along with their description
+
+| output         | description                                               |
+|----------------|-----------------------------------------------------------|
+| `domain_proof` | interpolated value of `aws_route53_record.ownership.name` |
```

Comment:
Is naming correct here? Should it be `domain_proof`/`ownership.name`?

Reply:
Updated!